src/pages/Main/SubNav/SubNav.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { Link, useLocation } from 'react-router-dom';
+import './SubNav.scss';
+
+const PAGES_NAME_LIST = [
+  { id: 1, path: '/', name: 'HOME' },
+  { id: 2, path: '/1', name: 'COMMUNITY' },
+  { id: 3, path: '/2', name: 'TRAINER MATCHING' },
+  { id: 4, path: '/3', name: 'SUBSCRIBE' },
+  { id: 5, path: '/login', name: 'LOGIN' },
+  { id: 6, path: '/5', name: 'DETAIL INFORMATION' },
+];
+
+const SubNav = () => {
+  const location = useLocation();
+  const nowPath = location.pathname;
+
+  const findPathName = PAGES_NAME_LIST => {
+    if (PAGES_NAME_LIST.path === nowPath) {
+      return true;
+    }
+  };
+
+  const path = PAGES_NAME_LIST.find(findPathName);
+  const pathName = path.name;
+
+  if (nowPath !== '/' && nowPath !== '/login') {
+    return (
+      <div className="SubNav">
+        <div className="categoryWrap">
+          <h1>{pathName}</h1>
+          <ul className="goPageList">
+            <li>
+              <Link to="/">HOME</Link>
+            </li>
+            <li>/ {pathName}</li>
+          </ul>
+        </div>
+      </div>
+    );
+  }
+};
```

Comment:
- exceptional path를 구현하고자 하신 거라면 아래처럼 구현 하면 될 것 같습니다.
- 다만, 실제 router의 path와 PAGES_NAME_LIST는 연결되어있어야 하는 데이터임에도 연결되지 않은 상태이기 때문에 더 꼼꼼한 관리가 필요합니다.
```suggestion
const EXCEPTIONAL_PATHS = ['/', '/login'];

const SubNav = () => {
  const { pathname } = useLocation();

  const isExceptionalPath = EXCEPTIONAL_PATHS.some(exceptionalPath => exceptionalPath === pathname);
  const pageName = PAGE_NAME_LIST.find(({ path }) => path === pathname)?.name;
  
  if(isExceptionalPath) return null;

  return (
    <div className="SubNav">
      <div className="categoryWrap">
        <h1>{pathName}</h1>
        <ul className="goPageList">
          <li>
            <Link to="/">HOME</Link>
          </li>
          <li>/ {pathName}</li>
        </ul>
      </div>
    </div>
  );
};
```